README.md

```diff
@@ -60,17 +60,22 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
-## Configuration
```

Comment:
We should keep this configuration section, possibly at the end.

Reply:
Weird.  I did copy it to the end.  I'm not sure why it's not there.  I'll re-push with your feedback.